server/image_pull.go

```diff
@@ -272,7 +279,19 @@ func tryIncrementImagePullFailureMetric(ctx context.Context, img string, err err
 	}
 }
 
-func decodeDockerAuth(s string) (user, password string, _ error) {
+func tryRecordSkippedMetric(ctx context.Context, name, digest string) {
+	layer := fmt.Sprintf("%s@%s", name, digest)
+	log.Debugf(ctx, "Skipped layer %s", layer)
+
+	counter, err := metrics.CRIOImageLayerReuse.GetMetricWithLabelValues(layer)
+	if err != nil {
+		log.Warnf(ctx, "Unable to write image layer reuse metrics: %v", err)
+	} else {
+		counter.Inc()
+	}
+}
+
+func decodeDockerAuth(s string) (user, password string, err error) {
```

Comment:
why'd this change to err?

Reply:
That’s the rest of the merge conflict resolution. I’ll revert it.